src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { InteractionResponseType, InteractionType } from "discord-interactions";
 import { Router } from "itty-router";
```

Comment:
Should be top import

Reply:
import { Router } from "itty-router";  I see it as 2nd import. As it is external it should be on the top. If it is already then please ignore my comment

Reply:
Even discord-interactions is a npm package